developwithpurpose/front-end-developer-exercise
# Riley mathews challenge attempt

# proposed changes
Added DOM structure for the web page with styling as specified in mockups. If javascript is enabled friend information will be loaded and displayed for each baby step. jQuery was added as a dependency through a CDN script tag added to the html page.

# steps to test
1. pull down code and checkout branch `riley-mathews-challenge-attempt`
1. run `npm start` to load up development server
1. navigate to `localhost:8080` if server is running on default port
1. Web page should be loaded with no issues. Clicking the nav items should navigate to the baby step and change the nav item clicked to blue while changing the previous active nav item to grey.
1. Disable javascript in your browser
1. The web page should still function normally. The only changes should be that friends information is no longer loaded, and the navigation is no longer responsive. 

# unfinished tasks
The animation is a css property that is not supported in most browsers (see readme). With more time a jquery animation can be written to simulate that behavior across more browsers, as well as add a scrolling bar to the navigation menu.
